### PR TITLE
Changed TypeConverters priority. Exist TypeConverter performs first, …

### PR DESCRIPTION
**Pull request by dkaera**
Changed TypeConverters priority. Exist TypeConverter performs first, even the column/model field type is a supported SQL type.
Fixes issue #735

### CODE REVIEW

**Review by memoriasIT — APPROVED** [review]

File: example/lib/main.dart [hunk 1]
```diff
@@ -65,7 +65,8 @@ class TasksWidgetState extends State<TasksWidget> {
           PopupMenuButton<int>(
             itemBuilder: (context) {
               return List.generate(
-                TaskType.values.length + 1, //Uses increment to handle All types
+                TaskStatus.values.length +
```
[the remark this answers is not in the document]
Comment (dkaera) [reply example/lib/main.dart:68]: auto-formatting moved it to a new line
it makes sense to remove the comment at the end of line

**Comment by dkaera** [comment]
> Hello, when will the new version be released

Thomas fell ill, so we're waiting for him to recover.

**Comment by zhangruiyu** [comment]
I'm sorry to bother you again. When can we release the new version? It's been 4 months since the last update. Sorry to bother you again

**Comment by dkaera** [comment]
> I'm sorry to bother you again. When can we release the new version? It's been 4 months since the last update. Sorry to bother you again

Yes, we're in the same boat. I'm waiting for a review too, but there haven't been any updates.

**Review by ThomasMiddel — APPROVED** [review]

**Pull request merged**